styles/style.css

```diff
@@ -63,10 +63,28 @@ p {
 .navigation__list {
   display: flex;
   list-style: none;
+  }
+}
+
+.nav1 {
+  margin: 30px;
+}
+
+ .nav2 {
+  margin: 30px;
+}
+                                        /* added new classes in order to fix navigation spacing issues */
+.nav3 {
+  margin: 30px;
+}
+
+.nav4 {
+  margin: 30px;
 }
 
 .navigation__item {
-  padding: 0.5rem 0;
+  display: flex;
```

Comment:
This won't need a display: flex because you are not using any flex property here.